src/ripple/json/json_value.h

```diff
@@ -284,13 +270,6 @@ class Value
     /// \post type() is unchanged
     void clear ();
 
-    /// Resize the array to size elements.
-    /// New elements are initialized to null.
-    /// May only be called on nullValue or arrayValue.
-    /// \pre type() is arrayValue or nullValue
-    /// \post type() is arrayValue
-    void resize ( UInt size );
-
     /// Access an array element (zero based index ).
     /// If the array contains less than index element, then null value are inserted
     /// in the array so that its size is index+1.
```

Comment:
Another small part that could be improved is `append`, it only has one overload -> `append(Value const&)` but I've seen pieces of code that call it as `append(std::move(val))` meaning they're expecting an overload like `append(Value&&)`

That's declared on line 292, but I can't put a comment there :)

Reply:
+1  This would actually get used a lot, right within the json lib itself.  Adding such a signature with `= delete` gives a preview of the first places the compiler finds where to use it.

Reply:
Coming Soon https://github.com/vinniefalco/json

Reply:
Right, @vinniefalco, I'm aware that's on the horizon.  And, once it's available, I anticipate we'll adopt it.  So I'm intentionally limiting these changes to be focused on stability and removing broken / unused / untested interfaces.